--- libindicate-12.10.1/Makefile.am~	2012-07-11 19:59:02.000000000 +0200
+++ libindicate-12.10.1/Makefile.am	2014-05-28 08:37:30.114776500 +0200
@@ -12,7 +12,6 @@
 	libindicate \
 	$(LIBINDICATEGTK_SUBDIR) \
 	bindings \
-	examples \
 	$(DOC_SUBDIR)
 
 if WANT_TESTS
--- libindicate-12.10.1/configure.ac~	2012-08-22 06:52:22.000000000 +0200
+++ libindicate-12.10.1/configure.ac	2014-05-28 08:53:09.540327973 +0200
@@ -227,7 +227,7 @@
 AC_MSG_RESULT($PYGTK_DEFSDIR)
 
 AC_MSG_CHECKING(for pygtk codegen)
-PYGTK_CODEGEN="$PYTHON `$PKG_CONFIG --variable=codegendir pygtk-2.0`/codegen.py"
+PYGTK_CODEGEN="$PYTHON `$PKG_CONFIG --variable=codegendir pygtk-2.0`/codegen.pyc"
 AC_SUBST(PYGTK_CODEGEN)
 AC_MSG_RESULT($PYGTK_CODEGEN)
 ])
--- libindicate-12.10.1/configure.ac~	2014-05-28 08:53:35.000000000 +0200
+++ libindicate-12.10.1/configure.ac	2014-05-28 08:56:46.084420978 +0200
@@ -216,7 +216,7 @@
 )
 
 # libpyglib-2.0-python* doesn't have a pkgconfig unfortunately
-INDICATE_PYTHON_LIBS+=" -lpyglib-2.0-python$PYTHON_VERSION"
+INDICATE_PYTHON_LIBS+=" -lpyglib-2.0-python"
 
 AC_SUBST(INDICATE_PYTHON_CFLAGS)
 AC_SUBST(INDICATE_PYTHON_LIBS)
